rabbitmq/docker-compose-smile.yml

```diff
@@ -388,6 +392,72 @@ services:
     - TWITTER_CONSUMER_SECRET=${TWITTER_CONSUMER_SECRET}
     restart: unless-stopped
 
+  clowder-list:
+    image: socialmediamacroscope/clowder_list:0.1.0
+    depends_on:
+      - rabbitmq
+    command: python3 ./rabbitmq_handler.py
+    environment:
+      - HOST_IP=${HOST_IP}
+      - RABBITMQ_HOST=${RABBITMQ_HOST}
+      - CLOWDER_BASE_URL=${CLOWDER_BASE_URL}
+      - CLOWDER_GLOBAL_KEY=${CLOWDER_GLOBAL_KEY}
+      - QUEUE_NAME=lambda_list_clowder
+    restart: unless-stopped
+
+  clowder-create-space:
+    image: socialmediamacroscope/clowder_create_space:0.1.0
+    depends_on:
+      - rabbitmq
+    command: python3 ./rabbitmq_handler.py
+    environment:
+      - HOST_IP=${HOST_IP}
+      - RABBITMQ_HOST=${RABBITMQ_HOST}
+      - CLOWDER_BASE_URL=${CLOWDER_BASE_URL}
+      - QUEUE_NAME=clowder_create_space
+    restart: unless-stopped
+
+  clowder-create-dataset:
+    image: socialmediamacroscope/clowder_create_dataset:0.1.0
+    depends_on:
+      - rabbitmq
+    command: python3 ./rabbitmq_handler.py
+    environment:
+      - HOST_IP=${HOST_IP}
+      - RABBITMQ_HOST=${RABBITMQ_HOST}
+      - CLOWDER_BASE_URL=${CLOWDER_BASE_URL}
+      - QUEUE_NAME=lambda_invoke_clowder
+    restart: unless-stopped
+
+  clowder-create-collection:
+    image: socialmediamacroscope/clowder_create_collection:0.1.0
+    depends_on:
+      - rabbitmq
+    command: python3 ./rabbitmq_handler.py
+    environment:
+      - HOST_IP=${HOST_IP}
+      - RABBITMQ_HOST=${RABBITMQ_HOST}
+      - CLOWDER_BASE_URL=${CLOWDER_BASE_URL}
+      - QUEUE_NAME=clowder_create_collection
+    restart: unless-stopped
+
+  clowder-upload-file:
+    image: socialmediamacroscope/clowder_upload_file:0.1.0
+    depends_on:
+      - rabbitmq
+    command: python3 ./rabbitmq_handler.py
+    environment:
+      - HOST_IP=${HOST_IP}
```

Comment:
I don't see if HOST_IP is needed in the code. As well as MINIO_PUBIC_ACCESS_URL. Also, it looks to me that HOST_IP is not being used in other clowder stuff